src/Universe.cpp

```diff
@@ -1214,6 +1214,41 @@ void Lattice::setUniverses(int num_z, int num_y, int num_x,
 
 
 /**
+ * @brief Update the Universe in a particular Lattice cell.
+ * @details This method may only be used after an array of Universe
```

Comment:
I think I should have commented on Universe being plural here instead of in `docstring.i`